news/_posts/2021-03-02-archived-modules.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Latest news about modules"
+subtitle: "Get some details about the archived modules"
+date:   2021-03-02 11:12:00
+authors:  [ MarionFrancois ]
+icon: icon-puzzle
+tags: 
+ - modules
+ - news
+---
+
+The product team recently did a survey of 600 users to find out which modules were essential and which they used the least.
+
+Following this survey, the product team and the developer team decided to maintain a smaller set of modules that are really important for the users, rather than attempting to keep all modules afloat which results in a poor maintenance of each and a constant struggle to update everything.
+
+The maintainers team resources are limited, and there are too many free and open source modules on the PrestaShop organization to be able to maintain them all. Some of these modules have not been given a new release for years.
+
+This is why the following set of modules, reported in the survey among the least used have been archived: statslive, ps_advertising, ps_feeder, ps_legalcompliance, ps_productinfo, ps_rssfeed, pscleaner, trackingfront, watermark, dateofdelivery, newsletter, prestafraud, vatnumber, ps_carriercomparison, referralprogram, ps_healthcheck, loyalty, gapi
+
+
+These modules have not been deleted, the code is still on GitHub, and you can still fork and use them on your store.
```

Comment:
```suggestion
The repositories of these modules have archived. This means that it will not be deleted and they will remain available on GitHub, but no further work will happen on these GitHub repositories. Developers can still fork them and use them.
```